sql/core/benchmarks/AggregateBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,143 @@
+================================================================================================
+aggregate without grouping
+================================================================================================
+
+OpenJDK 64-Bit Server VM 11.0.4+11-LTS on Linux 3.10.0-862.3.2.el7.x86_64
```

Comment:
Just in case, `3.10` kernel (if it is version of Linux kernel) is 2 years old. Are we sure we are interested in results on this old version?

Reply:
@MaxGekk . This is Java comparison, not OS comparison. We didn't care Mac OS benchmark result either, do we?

Reply:
I think the point is that the benchmark ratios could be different on another OS, but, I think this is fine. Eventually we can try benchmarking consistently on a later distro / 4.x kernel